### PR TITLE
Task/871 create get raster resolution

### PR DESCRIPTION
**Pull request by rasrocha**

### CODE REVIEW

File: src/coresyf_image_crop/coresyf_image_crop.py [hunk 1]
```diff
@@ -101,6 +101,24 @@ def apply_buffer_to_polygon(polygon_extent, buffer, crs_polygon, crs_buffer):
...
+    '''
+    gdal_raster = gdal.Open(raster_path)
+    transform = gdal_raster.GetGeoTransform()
+    return transform[1]
+
+
+def convert_buffer_units(raster_resolution, buffer):
+    '''
+    Converts the units of the buffer (given in pixels) to the units of the
+    image (given either in meters/pixel or degrees/pixel)
```
Comment (rmcapote) [inline src/coresyf_image_crop/coresyf_image_crop.py]: Change to:
"(given either in meters or degrees)"

File: src/coresyf_image_crop/tests/test_image_crop.py [hunk 2]
```diff
@@ -54,3 +53,7 @@ def test_apply_buffer_to_polygon(self):
                                                       crs_buffer)
 
         self.assertEqual(polygon_with_buffer.GetEnvelope(), expected_envelope)
+
+    def test_get_raster_resolution(self):
+            resolution = get_raster_resolution(self.test_image)
+            self.assertGreater(abs(resolution), 0)
```
Comment (rmcapote) [inline src/coresyf_image_crop/tests/test_image_crop.py]: Correct this line to:
'self.assertEqual(resolution, 180)'

File: src/coresyf_image_crop/coresyf_image_crop.py [hunk 3]
```diff
@@ -101,6 +101,24 @@ def apply_buffer_to_polygon(polygon_extent, buffer, crs_polygon, crs_buffer):
...
 
 
+def get_raster_resolution(raster_path):
+    '''
+    Retrieves the resolution of the raster located at raster_path,
+    in meters/pixel or degress/pixel.
+    '''
+    gdal_raster = gdal.Open(raster_path)
+    transform = gdal_raster.GetGeoTransform()
+    return transform[1]
```
Comment (rmcapote) [inline src/coresyf_image_crop/coresyf_image_crop.py]: Please ensure that the resolution is always a positive value.

**Review by rmcapote — CHANGES_REQUESTED** [review]
Please perform the suggested corrections.

File: src/coresyf_image_crop/coresyf_image_crop.py (hunk 1 above)
Comment (rasrocha) [reply src/coresyf_image_crop/coresyf_image_crop.py]: Done.

File: src/coresyf_image_crop/tests/test_image_crop.py (hunk 2 above)
Comment (rasrocha) [reply src/coresyf_image_crop/tests/test_image_crop.py]: Done.

File: src/coresyf_image_crop/coresyf_image_crop.py (hunk 3 above)
Comment (rasrocha) [reply src/coresyf_image_crop/coresyf_image_crop.py]: Done.

**Pull request merged**